test/lib/BedrockTester.cpp

```diff
@@ -553,4 +553,38 @@ int BedrockTester::waitForPort(int port) {
     return 1;
 }
 
+// get the output of a "Status" command from the command port
+STable BedrockTester::getStatus( bool control ) {
+    // FYI: sometimes the status command doesn't return with every key/value expected
+    SData request( "Status" );
+    string response = executeWaitVerifyContent( request, "200", control );
+    STable responseTable = SParseJSONObject(response);
+    return responseTable;
+}
+
+// get the value of a particular term from the output of a "Status" command
+string BedrockTester::getStatusTerm( string term, bool control ) {
+    STable responseTable;
+
+    // FYI: sometimes the status command doesn't return with every key/value expected
+    // loop until you find it
+    while (!responseTable.count(term)) {
```

Comment:
this could use a retry counter to prevent from looping forever for a typo'd term or removed term.